src/Sulu/Bundle/ContentBundle/Command/ValidatePagesCommand.php

```diff
@@ -71,6 +67,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $structures[] = $structure->getKey();
         }
 
+        $availableStructures = array();
+        foreach ($structureProvider->getStructures($webspaceKey) as $structure) {
+            $availableStructures[] = $structure->getKey();
```

Comment:
Would rename that to `$webspaceStructureKeys`, the current name implies that `Structure` objects are saved.